### PR TITLE
Remove metrics feature flag

### PR DESCRIPTION
**Pull request by kathy-t**
**Description**
Removes the metrics feature flag from the UI.

The versions tab is crowded if both the Verified Platforms and Metrics columns are present, which creates a scroll bar to access the Actions column. Previously, the Verified Platforms column was hidden if the metrics feature flag was on.

This PR turns the Verified Platforms column into a Verified column where the value is a check mark if the version is verified and moves the Verified Platforms information into the Info dialog.

**Before:**
image

**After:**
Screenshot 2024-01-16 at 12-07-33 Dockstore Workflow github com_DataBiosphere_topmed-workflows_UM_variant_caller_wdl

In the Info dialog, if the version is Verified, then Verified Platforms are displayed.
Screenshot 2024-01-16 at 12-11-03 Dockstore Workflow github com_alexanderhsieh_test_read_tsv_test_read_tsv github com_DataBiosphere_topmed-workflows_UM_variant_caller_wdl

If the version is not Verified, then Verified Platforms are not shown.
Screenshot 2024-01-16 at 12-11-16 Dockstore Workflow github com_alexanderhsieh_test_read_tsv_test_read_tsv github com_DataBiosphere_topmed-workflows_UM_variant_caller_wdl

<details>
<summary>Old PR description</summary>
The versions tab is pretty crowded now and the Actions button is hidden unless scrolled to:
Screenshot 2024-01-12 at 10-22-52 Dockstore Workflow github com_theiagen_terra_utilities_Concatenate_Column_Content

Screenshot 2024-01-12 at 10-23-08 Dockstore Workflow github com_theiagen_terra_utilities_Concatenate_Column_Content

Previously we hid the Verified Platforms column if the metrics flag was enabled. Should we permanently hid the Verified Platforms column in favour of the metrics column?

</details>

**Review Instructions**
During these steps, ensure that there is no "preview" banner.

Verify that you can:
- View the metrics facet on the search page
- select a workflow with metrics from the search page
- Click on the versions tab and verify that the metrics column is displayed
- Select a version with metrics and go to the Metrics tab
- Verify that the metrics are displayed

**Issue**
[SEAB-6090](https://ucsc-cgl.atlassian.net/browse/SEAB-6090)

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.
- [x] Check whether this PR disables tests. If it legitimately needs to disable a test, create a new ticket to re-enable it in a specific milestone. 

[SEAB-6090]: https://ucsc-cgl.atlassian.net/browse/SEAB-6090?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by denis-yuen — COMMENTED** [review]
discussion

**Comment by svonworl** [comment]
"The versions tab is pretty crowded now and the Actions button is hidden unless scrolled to:"

There's an easy style fix that we could put into this PR that would make the width of the subtabs ("Info", "DAG", "Actions") adapt to the available space, which would unhide the "Actions" button in most cases.  I used it in the PR that added the "Archived" header to the entries accordion, I can look it up if we think it's a good idea...

**Comment by coverbeck** [comment]
> Previously we hid the Verified Platforms column if the metrics flag was enabled. Should we permanently hid the Verified Platforms column in favour of the metrics column?

I feel like as the quickest and safest fix, we could remove it for this release. There are only 16 workflows in prod that have the verified platforms set.

Moving it and other stuff into the Info dialog works as well, although I think it could wait for 1.15.x or 1.16 if we think it's too much work.

I can't quite visualize Steve's idea, but that sounds reasonable too.

Much longer term, each column takes up a fair amount of space, because of the header, so perhaps we could could have one column that has a bunch of boolean values in it, represented by icons, if that makes sense.

**Comment by svonworl** [comment]
> "The versions tab is pretty crowded now and the Actions button is hidden unless scrolled to:"
> 
> There's an easy style fix that we could put into this PR that would make the width of the subtabs ("Info", "DAG", "Actions") adapt to the available space, which would unhide the "Actions" button in most cases. I used it in the PR that added the "Archived" header to the entries accordion, I can look it up if we think it's a good idea...

In https://github.com/dockstore/dockstore-ui2/pull/1852, I added the following code, which makes the width of the "accordion" tabs adaptive, and sets some padding to ensure that they never get too narrow:

https://github.com/dockstore/dockstore-ui2/blob/develop/src/app/shared/styles/my-entry-sidebar.scss#L70-L73

Would love to see us do something similar for the main entry info tabs, I advocated for it a few months ago...

**Review by coverbeck — APPROVED** [review]
Approving, but if you're going to change it based on versions table discussion, I'd be interested in taking another look.

**Comment by denis-yuen** [comment]
> Much longer term, each column takes up a fair amount of space, because of the header, so perhaps we could could have one column that has a bunch of boolean values in it, represented by icons, if that makes sense.

Also like this one. I feel like there are a lot of ideas :grin:

**Comment by svonworl** [comment]
> > "The versions tab is pretty crowded now and the Actions button is hidden unless scrolled to:"
> > There's an easy style fix that we could put into this PR that would make the width of the subtabs ("Info", "DAG", "Actions") adapt to the available space, which would unhide the "Actions" button in most cases. I used it in the PR that added the "Archived" header to the entries accordion, I can look it up if we think it's a good idea...
> 
> In #1852, I added the following code, which makes the width of the "accordion" tabs adaptive, and sets some padding to ensure that they never get too narrow:
> 
> https://github.com/dockstore/dockstore-ui2/blob/develop/src/app/shared/styles/my-entry-sidebar.scss#L70-L73
> 
> Would love to see us do something similar for the main entry info tabs, I advocated for it a few months ago...

Now, I realize that I got a bit mixed up and, in the above, was thinking about the main entry page tab bar.  But, maybe something like the above could be used similarly in the Versions table, too.

**Comment by kathy-t** [comment]
The PR description is updated with the newest change where the Verified Platforms info is moved into the Info dialog. I didn't remove the Verified Platforms column completely - I changed it to a boolean "Verified" column, which is short enough that it doesn't introduce a scroll bar when the metrics column is present.

Screenshots are in the PR description.

**Comment by denis-yuen** [comment]
> In the Info dialog, if the version is Verified, then Verified Platforms are displayed.

:+1:

**Review by denis-yuen — APPROVED** [review]

**Review by svonworl — APPROVED** [review]

**Review by hyunnaye — APPROVED** [review]

File: src/app/workflow/versions/versions.component.html [hunk 1]
```diff
@@ -162,11 +162,11 @@
 
     <ng-container matColumnDef="verified">
       <th scope="col" mat-header-cell *matHeaderCellDef [matTooltip]="verifiedVersionTooltip">
-        Verified Platforms
+        Verified
         <a class="ds-green" [href]="verifiedLink" target="_blank" rel="noopener noreferrer"><mat-icon>info</mat-icon></a>
       </th>
       <td mat-cell *matCellDef="let version">
-        <span matTooltip="{{ getVerifiedSource(version.name) }}">{{ version.id | verifiedPlatforms: verifiedVersionPlatforms }}</span>
+        <mat-icon *ngIf="version?.verified">check</mat-icon>
```
Comment (coverbeck) [inline src/app/workflow/versions/versions.component.html:169]: Nit: I think the question mark is redundant, but no need to change -- the file is already inconsistent with its usage of it.

**Review by coverbeck — APPROVED** [review]

**Pull request merged**